src/Styleguide/Pages/Artist/Routes/RelatedArtists/RelatedArtistsRefetchContainer.tsx

```diff
@@ -192,13 +186,7 @@ export const RelatedArtistsRefetchContainer = createRefetchContainer(
             }
             edges {
               node {
-                name
-                image {
-                  cropped(width: 400, height: 300) {
-                    url
-                  }
-                }
-                formatted_nationality_and_birthday
+                ...ArtistCard_artist @relay(mask: false)
```

Comment:
@alloy another one of these..

Reply:
I think the above is the cause of this issue https://github.com/artsy/reaction/pull/861/files#r197541076